build.gradle.kts

```diff
@@ -232,4 +235,52 @@ openApiMerger {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+// Dependency analysis active if property "dependency.analysis" is set. Possible values are <'fail'|'warn'|'ignore'>.g po
+if (project.hasProperty("dependency.analysis")) {
+    apply(plugin = "org.eclipse.dataspaceconnector.dependency-rules")
+    configure<org.eclipse.dataspaceconnector.gradle.DependencyRulesPluginExtension> {
+        severity.set(project.property("dependency.analysis").toString())
+    }
+    apply(plugin = "com.autonomousapps.dependency-analysis")
+    configure<com.autonomousapps.DependencyAnalysisExtension> {
+        // See https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin
+        issues {
+            all { // all projects
+                onAny {
+                    severity(project.property("dependency.analysis").toString())
+                    exclude(
+                        "org.jetbrains:annotations",
+                        "com.fasterxml.jackson.datatype:jackson-datatype-jsr310",
+                        "com.fasterxml.jackson.core:jackson-core",
+                        "com.fasterxml.jackson.core:jackson-annotations",
+                        "com.fasterxml.jackson.core:jackson-databind",
+                        "com.fasterxml.jackson.datatype:jackson-datatype-jsr310",
+                    )
+                }
+                onUnusedDependencies {
+                    exclude(
+                        "com.github.javafaker:javafaker",
+                        "org.assertj:assertj-core",
+                        "org.junit.jupiter:junit-jupiter-api",
+                        "org.junit.jupiter:junit-jupiter-params",
+                        "org.mockito:mockito-core",
+                    )
+                }
+                onIncorrectConfiguration {
+                    exclude(
+                        // some common dependencies are intentionally exported by core:base for simplicity
+                        "com.squareup.okhttp3:okhttp",
+                        "net.jodah:failsafe",
+                        // public methods annotated with @WithSpan
+                        "io.opentelemetry:opentelemetry-extension-annotations",
```

Comment:
with rc04 should be able to use excludeAnnotations instead https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/610

Reply:
Done.